src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -422,31 +415,23 @@ private void finalizeCurrentVariant(final List<VariantContext> unmergedCalls,
             unmergedCalls.add(createRefSiteVariantContextWithGQ(missingSample, contig, start, refAllele, MISSING_CONF_THRESHOLD));
         }
 
-        // we only need to retain the NonRefSymbolicAllele if we are using gnarly
-        boolean removeNonRefSymbolicAllele = this.disableGnarlyGenotyper;
-
         final VariantContext mergedVC = variantContextMerger.merge(
                 unmergedCalls,
                 new SimpleInterval(contig, (int) start, (int) start),
                 refAllele.getBases()[0],
-                removeNonRefSymbolicAllele,
+                true,
                 false,
                 true);
 
         ReferenceContext referenceContext = new ReferenceContext(refSource, new SimpleInterval(mergedVC));
 
         VariantContext annotatedVC = annotationEngine.annotateContext(mergedVC, new FeatureContext(), referenceContext, null, a -> true);
 
-        final VariantContext genotypedVC = this.disableGnarlyGenotyper ? annotatedVC : gnarlyGenotyper.finalizeGenotype(annotatedVC);
-
-        // Gnarly will indicate dropping a site by returning a null from the finalizeGenotype method
-        if (!this.disableGnarlyGenotyper && genotypedVC == null) {
-            return;
-        }
+        final VariantContext genotypedVC = annotatedVC;
```

Comment:
you could name the variable in the previous line genotypedVC and then get rid of this line